libbeat/publisher/pipeline/stress/data/placeholder

```diff
@@ -0,0 +1 @@
+// Do not remove
```

Comment:
Perhaps add a note on why it should not be removed.

Reply:
will change tests to create a temporary directories. Was just nice to have files available after tests (for inspection).

When removing the file, git will remove the directory -> tests would fail.